app/controllers/application_controller.rb

```diff
@@ -96,9 +82,7 @@ def process_index(base, serializer_class, opts: {}, default_sort: nil, allowed_s
 
   def index_meta(scope)
     meta = {
-      :total => scope.count,
-      :per_page => scope.per_page,
-      :page => scope.current_page
+      :total => scope.limit(nil).offset(nil).count
```

Comment:
Do we really need to unset the limit and offset to get the correct count?

Reply:
Yep:

```
[3] pry(main)> App.offset(5).offset(nil).count
   (2.1ms)  SELECT COUNT(*) FROM "apps"
=> 10
[4] pry(main)> App.offset(5).count
   (0.8ms)  SELECT COUNT(*) FROM (SELECT 1 AS one FROM "apps" OFFSET $1) subquery_for_count  [["OFFSET", 5]]
=> 5
```

The same goes for limit.